cql3/statements/alter_keyspace_statement.cc

```diff
@@ -113,9 +115,7 @@ cql3::statements::alter_keyspace_statement::prepare_schema_mutations(query_proce
                 service::topology_mutation_builder builder(ts);
                 builder.set_global_topology_request(service::global_topology_request::keyspace_rf_change);
                 builder.set_global_topology_request_id(this->global_req_id);
-                // TODO: need to correctly handle the generic 'replication_factor' tag in NTS
-                //       currently, if it's used, we're passing an empty map and some test.py testcases fail
```

Comment:
patch lacks motivation and some explanations on how it now "correctly handle the generic 'replication_factor' tag"

Reply:
I second that comment. Please, add patch comment with motivation description

Reply:
This patch doesn't support it, it ignores it. That being said, I provided an explanatory comment in this commit.
A note to myself: whether to support this tag or drop it is under discussion.

Reply:
New version of patch "Ignore NTS's replication_factor tag" has a commit message that includes the motivation hence the deletion of the TODO comment is justified and this thread can be resolved.